content/applications/inventory_and_mrp/quality/quality_reporting/quality_alerts_report.rst

```diff
@@ -0,0 +1,210 @@
+=====================
+Quality alerts report
+=====================
+
+In Odoo's *Quality* app, the quality alerts report displays key statistics about existing alerts.
+This includes the number of quality alerts in each stage, the number of alerts assigned to each
+quality team, the root causes of the issues that led to alert creation, and more.
+
+The report offers a number of options to help visualize the data, including customizable views and
+three different graph types. This allows users to extract deeper insights from their quality alert
+processes.
+
+Report dashboard
+================
+
+To access the quality alerts report dashboard, navigate to :menuselection:`Quality app --> Reporting
+--> Quality Alerts`.
+
+There are five main views that can be used to display quality alert data:
+
+- :ref:`Graph view <quality_alerts_report/graph-view>`
+- :ref:`Pivot view <quality_alerts_report/pivot-view>`
+- :ref:`Kanban view <quality_alerts_report/kanban-view>`
+- :ref:`List view <quality_alerts_report/list-view>`
+- :ref:`Calendar view <quality_alerts_report/calendar-view>`
+
+By default, the dashboard displays data in the graph view, using the bar chart graph type. To select
+a different view, click on the corresponding button in the top-right corner of the dashboard.
+
+.. figure:: quality_alerts_report/view-select.png
+   :align: center
+   :alt: The view select buttons on the quality alerts report dashboard.
+
+   The view select buttons on the quality alerts report dashboard. From left to right: Graph, Pivot,
+   Kanban, List, and Calendar.
+
+.. _quality_alerts_report/graph-view:
+
+Graph view
+----------
+
+To display quality alert data in graph view, click the :icon:`fa-area-chart` :guilabel:`(Graph)`
+button in the top-right corner of the dashboard.
+
+There are three chart types that can be used to visualize quality alert data:
+
+- :ref:`Bar chart <quality_alerts_report/bar-chart>`
+- :ref:`Line chart <quality_alerts_report/line-chart>`
+- :ref:`Pie chart <quality_alerts_report/pie-chart>`
+
+By default, graph view displays data using the bar chart graph type. To select a different graph
+type, click the corresponding button in the top-left corner of the dashboard.
+
+.. figure:: quality_alerts_report/graph-select.png
+   :align: center
+   :alt: The graph select buttons on the quality alerts report dashboard.
+
+   The graph select buttons on the quality alerts report dashboard. From left to right: Bar Chart,
+   Line Chart, and Pie Chart.
+
+.. _quality_alerts_report/bar-chart:
+
+Bar chart
+~~~~~~~~~
+
+To display quality alert data in a bar chart, click the :icon:`fa-bar-chart` :guilabel:`(Bar Chart)`
+button in the top-left corner of the dashboard.
+
+The x-axis of the bar chart lists root causes that led to the creation of one or more alerts (ex.
```

Comment:
```suggestion
The x-axis of the bar chart lists a specific :guilabel:`Root Cause` that led to the creation of one or more alerts (e.g.
```
It's my understanding that we're going with "e.g." in these parenthetical instances...
Also, since Root Cause appears on the screen, I believe a GUI should be applied.